addons/link_tracker/tests/test_link_tracker.py

```diff
@@ -162,3 +162,21 @@ def test_no_loop(self):
         self.assertRaises(UserError, self.env['link.tracker'].create, {'url': '?debug=1'})
         self.assertRaises(UserError, self.env['link.tracker'].create, {'url': '#'})
         self.assertRaises(UserError, self.env['link.tracker'].create, {'url': '#model=project.task&id=3603607'})
+
+    def test_absolute_url(self):
```

Comment:
Would be best kept in alphabetical order within means, to keep order of test execution (i.e. top of the file for this one)